Phaser/phaser-workshop.adoc

```diff
@@ -235,10 +248,13 @@ Voeg onderstaande code toe aan het einde van de functie 'sterPakken' zodat er me
 ----
 // wat moet er gebeuren als alle sterren zijn verzameld?
 if (sterren.countActive(true) === 0) {
+
+    // Zet alle sterren weer aan
     sterren.children.iterate(function (child) {
         child.enableBody(true, child.x, 0, true, true);
```

Comment:
Andere opmerking: voor de begrijpelijkheid kan het waarschijnlijk iets helpen als we de functie die als argument is gedeclareerd uit de functie trekken, en hem iets noemen als "maakSter"